src/drivers/frsky_telemetry/frsky_telemetry.c

```diff
@@ -266,18 +246,17 @@ static int frsky_telemetry_thread_main(int argc, char *argv[])
 			 * in order to apply a lowpass filter to baro pressure.
 			 */
 			static float last_baro_alt = 0;
-			bool sensor_updated;
-			orb_check(sensor_sub, &sensor_updated);
+			struct sensor_combined_s raw;
+			memset(&raw, 0, sizeof(raw));
```

Comment:
If you copy the struct there is no value in setting it to zero. It will be completely overwritten.